lib/class-wp-theme-json-schema-v0.php

```diff
@@ -0,0 +1,339 @@
+<?php
+/**
+ * Processes structures that adhere to the theme.json schema V0.
+ *
+ * @package gutenberg
+ */
+
+/**
+ * Class that encapsulates the processing of
+ * structures that adhere to the theme.json V0.
+ */
+class WP_Theme_JSON_Schema_V0 {
```

Comment:
I guess ideally, this should implement an "interface" that way new deprecated schemas would know what they have to implement.